cli/command/command.go

```diff
@@ -144,6 +144,18 @@ func ConvertOpenShiftCommand() cli.Command {
 				Name:  "deploymentconfig,dc",
 				Usage: "Generate a OpenShift DeploymentConfig object",
 			},
+			cli.StringFlag{
```

Comment:
@kadel We need it right now for the following usecases:

- point to development repo/branch
- if the remote origin repo url is of type ``git`` or ``ssh``, it needs the necessary keys to pull the repo, even if it's public. That's why openshift, in our current implementation, fails to pull from those repos.